packages/generator-common/src/file-writer/package-json.ts

```diff
@@ -33,12 +25,12 @@ export function packageJsonBase(
 ): Record<string, any> {
   return {
     name: options.npmPackageName,
-    version: options.version,
+    version: '1.0.0',
     description: options.description,
     homepage: 'https://sap.github.io/cloud-sdk/docs/js/getting-started',
     main: './index.js',
     types: './index.d.ts',
-    ...(options.license ? { license: options.license } : {}),
+    license: 'Apache-2.0',
```

Comment:
I think the default license should be 'UNLICENSED' cf https://stackoverflow.com/a/33755454 because we can't make that decision on behalf of our users.

Side-note: apache 2 license does not imply the source code is publicly available. a customer might still chose the apache license for their only-internally available source code.

Question: how does this align with the sap developer license that applies to the generated code?

Reply:
Ah `UNLICENSED` the unkown for the license field. I like.

Reply:
Regarding your question I would say this strongly depends on the system and service. If it is a custom service people can do what they want I would say. If not they can not publish this - in any way I would say people will never put clients in the public NPM either it is their private thing or it is a  SAP owned service then they would violate some IP thing.